relayer/src/config.rs

```diff
@@ -51,7 +51,7 @@ pub mod default {
     }
 
     pub fn tx_confirmation() -> bool {
-        true
+        false
```

Comment:
This should be documented explicitly in the release notes as it changes a default.

Reply:
I updated the changelog entry to explicitly state the change of default value